base/server/man/man8/pki-server-nuxwdog.8

```diff
@@ -37,10 +37,10 @@ to remove this file entirely, and have the server prompt for these passwords on
 startup.  This means of course that it will not be possible to start the CS
 instance unattended, including on server reboots.
 .PP
-\fBnuxwdog\fR is a daemon that will launch the CS instance and prompt the administrator
-for the relevant passwords.  These passwords will be cached securely in the kernel
-keyring.  If the CS instance crashes unexpectedly, \fBnuxwdog\fR will attempt to restart
-the instance using the cached passwords.
+\fBnuxwdog\fR is a script that is launched before the CS instance starts and prompts
+the administrator for the relevant passwords.  These passwords will be cached securely
+in the kernel keyring.  If the CS instance crashes unexpectedly, \fBsystemd\fR will attempt
+to restart the instance using the cached passwords.
 .PP
 CS instances need to be reconfigured to use \fBnuxwdog\fR to start.  Not only are changes
 required in instance configuration files, but instances need to use a different
```

Comment:
does the new nuxwdog still uses an "instance configuration files" as mentioned here?

Reply:
IIUC, yes. Contents of CS.cfg are modified -- PasswordClass and PasswordFile

Reply:
ok I thought I was reviewing nuxwdog man pages.  So, looks like nuxwdog man page should be removed.  Did you do that?  I might have missed it.

Reply:
as per IRC conversation, the instance conf here refers to instance's CS.cfg. 

Though nuxwdog.conf is created from PKI, its man page is included in the original nuxwdog repo. Here: https://github.com/dogtagpki/nuxwdog/blob/master/src/doc/nuxwdog.1